package.json

```diff
@@ -10,6 +10,7 @@
     "@typespec/compiler": "0.48.0",
     "@typespec/http": "0.48.0",
     "@typespec/openapi": "0.48.0",
+    "@typespec/openapi3": "0.48.0",
```

Comment:
@glecaros: Do you think we will need to add this dependency to the root `package.json`?  If so, our team will need to discuss this before it can be added.

CC: @timotheeguerin, @markcowl, @lmazuel

Reply:
We can probably remove it once we publish this as I'm only using this as help while making changes. I'm using OpenAPI 3 as the generator handles shared route better than v2.